README.md

```diff
@@ -9,6 +9,7 @@ Similar to Jest's `expect`, but with type-awareness. Gives you access to a numbe
 
 ```ts
 import {expectTypeOf} from 'expect-type'
+import {test} from 'vitest'
```

Comment:
Hm.. If this example aims to demonstrate how `expect-type` "can be used in your existing test files", I guess following is what users would actually do:

```suggestion
import {expectTypeOf, test} from 'vitest'
```

Reply:
It's too confusing to introduce vitest this early. I'd rather get rid of the `test(...)` and just have some raw expectTypeOfs here.